### ISSUE TITLE
Missing discord-voice.c from Makefile in concord/src 

### ISSUE THREAD

**Issue by Mentosz888** [Cogmasters/concord#186]
I tried to use the discord_voice_join() function in my project, but at compilation I got a undefined refrence error to this function. From looking trough the src files, I realized the discord-voice.c is missing from the Makefile. Maybe I am unaware of some deprecation, but it seems to me, this is the only function that allows my bot to connect to a vc.

**Comment by jdeokkim** [comment]
Hello, have you tried using `make voice` when building this library?

**Comment by Mentosz888** [comment]
Damn, you are absoluetly right. I have missed this rule entirely. Thanks!

**Comment by Mentosz888** [comment]
Update on the situation, I got the voice to compile successfully. I've been looking at the internals of this project, and been testing the voice join and leave. For the usage of the discord_voice_shutdown() I extended the discord-voice.c file so I can get the vc by guild ID. There is an error in the gencodecs discord_update_voice_state_to_jsonb() function. 
Without modification, currently it send this JSON for a disconnect: {"op":4,"d":{"guild_id":"GUILD_ID","channel_id":"0","self_mute":false,"self_deaf":false}}
And it should send for a successfull disconnect: {"op":4,"d":{"guild_id":"GUILD_ID","channel_id":"null","self_mute":false,"self_deaf":false}}
